samples/bluetooth/central_otc/src/main.c

```diff
@@ -134,12 +140,13 @@ static void otc_btn_work_fn(struct k_work *work)
 				obj_data_buf[idx] = UINT8_MAX - (idx % UINT8_MAX);
 			}
 
+			last_checksum = crc32_ieee(obj_data_buf, size_to_write);
```

Comment:
I would suggest to move the `crc32_ieee` to a `bt_ots_client_calc_checksum` function so that the client application do not need to worry about how to calculate it correctly.

Reply:
I somewhat agree, but my point is that with this approach, the application needs to know exactly which algorithm to use to calculate the checksum (in this case, the one implemented by `crc32_ieee`, but if there was a `bt_ots_client_calc_checksum` function, that knowledge could (easily) be abstracted, making it easier to use the OTC. 

If you disagree, then we should at least (if not already) have a reference to `crc32_ieee` in the documentation.

Reply:
Add bt_ots_client_calc_checksum function as a wrapper to call crc32_ieee().